orbital4c/NuclearPotential.py

```diff
@@ -2,15 +2,16 @@
 import copy as cp
 from scipy.special import erf
 
-def CoulombPotential(position, center, charge):
+
+def PoCh(position, center , charge):
```

Comment:
Could you please use longer names? "PoCh" --> "CoulombPotential"
As I told you earlier, your future self will thank you!